person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_accessor :id, :name, :age
+
+  def of_age?
+    @age > 18
+  end
+
+  def can_use_services?
+    @age > 18 or @parent_permission == true
+  end
+  private :is_of_age?
+  public :can_use_services?
```

Comment:
- [ ] `is_of_age?` method is not defined in your class, I guess you wanted to assign` of_age` as a private method. Let's fix that
- [ ] You don't need to declare `can_use_service? `as a public method since it is not a private one, it is public by default.